### PR TITLE
[alpha.webkit.UncountedCallArgsChecker] Ignore calls to WTF's container methods

### PR DESCRIPTION
**Pull request by rniwa**
This PR makes the checker ignore / skip calls to methods of Web Template Platform's container types such as HashMap, HashSet, WeakHashSet, WeakHashMap, Vector, etc...

### CODE REVIEW

File: clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp [hunk 1]
```diff
@@ -25,6 +25,11 @@ using namespace ento;
 
 namespace {
 
+bool stringEndsWith(const std::string &str, const std::string &suffix) {
+  auto index = str.rfind(suffix);
+  return index != std::string::npos && str.size() - suffix.size() == index;
+}
```
Comment (haoNoQ) [inline clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp:31]: Consider `llvm::StringRef::ends_with()`.

Comment (rniwa) [reply clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp:31]: Oh, didn't realize we had that. I'll add a FIXME for making `safeGetName` return a `StringRef`.

Comment (rniwa) [reply clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp]: Fixed!

**Review by haoNoQ — APPROVED** [review]
Ooo this is much cleaner!

**Comment by rniwa** [comment]
Yeah, indeed. Thank you for the review!

**Pull request merged**